jennikate/sunnyglade-ratters
# Initial setup

# In this PR
Setup React app with webpack & babel, with basic eslint rules

## To test

- `npm i`
- `npm start`
- > you should see a page with a heading of 'Basic Setup'
